ongs/serializers.py

```diff
@@ -23,6 +25,11 @@ class Meta:
         ]
 
     def create(self, validated_data: dict):
+        cnpj = validated_data.get("cnpj", None)
+
+        if not check_cnpj_mask(cnpj):
```

Comment:
if `cnpj` is `None`, we will be passing it to the `check_cnpj_mask` function, and since the first thing we do in that function is to verify the length of the supposedly string passed, and its value would be `None`, it will raise an `TypeError`.

My suggestion would be to pass the `validated_data` to the `check_cnpj_mask` function and even raise the error there, if possible.

Reply:
Ok